Using a more updated Dockerimage

Gallimathias opened the pull request:
* After the build with debian 10 fails, this PR updates the whole thing to a current Alpine.

  Alpine for the reason that it is pleasantly slim for a CI server.

Gallimathias committed:
Using a more updated Dockerimage

* After the build with debian 10 fails, this PR updates the whole thing
  to a current Alpine.

  Alpine for the reason that it is pleasantly slim for a CI server.

nguy8tri reviewed (changes requested):
Thanks so much for making this PR! I also ran into the same issue using this GitHub action [here](https://github.com/UWCubeSat/found/actions/runs/16244813797/job/45866076074). I just suggested a change to make a consistent but also lasting change.

jmckenna commented:
@erclu can you approve this workflow (so we can see if this proposed change runs well?)  thanks.

Gallimathias committed:
Use Debian slim instead alpine

* Alpine is slimmer in itself but seems to be slower for the specific
  case, as a field test has shown.

  In addition, the adaptation is smaller, which should increase the
  acceptance of the PR

nguy8tri reviewed (approved):
Great job, thanks so much for doing this! @erclu we are now just waiting on your approval as well, please take your time.

jmckenna commented:
thanks all.  

PS. until a new version can be tagged with this change, pointing to master works well now: `erclu/check-crlf@master`
